src/api/resourses/users/UserController.js

```diff
@@ -71,7 +71,7 @@ class UserController {
             ]
           });
       })
-      .catch(error => {
+      .catch(err => {
         res.status(400).json({ status: 400, error: "something happerned" });
       });
```

Comment:
Your error message is not descriptive enough, it says `something happened` what happened because anything can happen.

Reply:
Ok Thanks its skipped my mind its database error

Reply:
Ok thanks